keyboards/keychron/q1/ansi_stm32l432/config.h

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2021 @ Keychron (https://www.keychron.com)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* key matrix pins */
+#define MATRIX_ROW_PINS { B5, B4, B3, A15, A14, A13 }
+#define MATRIX_COL_PINS { C14, C15, A0, A1, A2, A3, A4, A5, NO_PIN, NO_PIN, NO_PIN, NO_PIN, NO_PIN, NO_PIN, NO_PIN, NO_PIN }
```

Comment:
these should be moved to the info.json file. 

Also, the NO_PIN's should be removed

Reply:
Hi, It failed to compile when i removed the NO_PINs from matrix_pins definiton😂.

Reply:
Compile information to remind me that: keychron/q1/ansi_stm32l432: led_config: ansi_stm32l432.c: Number of columns in row 0 (16) does not match matrix (8) and that: keychron/q1/ansi_stm32l432: led_config: ansi_stm32l432.c: Number of columns in row 0 (16) does not match matrix (8)

Reply:
There is a higher level that has MATRIX_COLS and MATRIX_ROWS defined?  If so, that would be why

Reply:
This board uses a custom matrix; pins declared as `NO_PIN` are actually connected to a shift register.  The code is mostly the same as for other similar Keychron boards already in the repo (q10, q12, q3, q5, q6, q65, v1, v10, v3, v5, v6).